### PR TITLE
CM-985 *FUNDING_REQUEST_ACCEPTED* event's emails

### LINKED ISSUE #985: *Funding proposal approved* flow [ Email ]
event name: **_FUNDING_REQUEST_ACCEPTED_**

Send a corresponding email (by its template) to:
  - proposal owner (request to upload invoices by entering a link that we are attaching to the email)
  - common support
  - common owner

### PR DESCRIPTION
**Pull request by SobolevYuri**

### CODE REVIEW

File: functions/src/notification/email/templates/proposerFundingRequestAcceptedZeroAmount.ts [hunk 1]
```diff
@@ -30,7 +31,7 @@ const emailStubs = {
   },
 };
 
-export const userFundingRequestAcceptedZeroAmount = {
+export const proposerFundingRequestAcceptedZeroAmount = {
```
Comment (ariel-dao) [inline functions/src/notification/email/templates/proposerFundingRequestAcceptedZeroAmount.ts]: funding request can't be zero amount, I think it's safe to remove

File: functions/src/notification/email/templates/commonOwnerFundingRequestAcceptedUnknown.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+//TODO
+
+import { testFlag } from '../../helpers';
+
+const template = `
+commonOwnerFundingRequestAcceptedUnknown moked-up email body
```
Comment (ariel-dao) [inline functions/src/notification/email/templates/commonOwnerFundingRequestAcceptedUnknown.ts]: what is the difference between foreign and unknown? can we remove unknown and just use foreign ?

**Review by ariel-dao — CHANGES_REQUESTED** [review]
added comments

File: functions/src/notification/email/templates/commonOwnerFundingRequestAcceptedUnknown.ts (hunk 2 above)
Comment (SobolevYuri) [reply functions/src/notification/email/templates/commonOwnerFundingRequestAcceptedUnknown.ts]: I think we can, and if it'll be needed we'll add this back then. But maybe the foreign (english) version would be enough for us.

**Review by SobolevYuri — COMMENTED** [review]
resolved

**Review by ariel-dao — APPROVED** [review]

**Pull request merged**